src/redist/targets/GenerateLayout.targets

```diff
@@ -46,11 +49,13 @@
 
       <!-- Use the "x64" Rid when downloading Linux shared framework 'DEB' installer files. -->
       <SharedFrameworkInstallerFileRid>$(CoreSetupRid)</SharedFrameworkInstallerFileRid>
-      <SharedFrameworkInstallerFileRid Condition=" '$(IsDebianBaseDistro)' == 'true' OR '$(IsRPMBasedDistro)' == 'true' ">$(Architecture)</SharedFrameworkInstallerFileRid>
+      <SharedFrameworkInstallerFileRid Condition=" '$(IsDebianBaseDistro)' == 'true' OR '$(IsRPMBasedDistro)' == 'true' ">$(InstallerTargetArchitecture)</SharedFrameworkInstallerFileRid>
 
       <!-- Use the "x64" Rid when downloading Linux runtime dependencies Debian package. -->
       <RuntimeDepsInstallerFileRid>$(CoreSetupRid)</RuntimeDepsInstallerFileRid>
       <RuntimeDepsInstallerFileRid Condition=" '$(IsDebianBaseDistro)' == 'true' ">$(Architecture)</RuntimeDepsInstallerFileRid>
+      <RuntimeDepsInstallerFileRid Condition=" '$(IsRPMBasedDistro)' == 'true' And '$(UsePortableLinuxSharedFramework)' != 'true' ">$(OSName)-$(InstallerTargetArchitecture)</RuntimeDepsInstallerFileRid>
+      <RuntimeDepsInstallerFileRid Condition=" '$(CoreSetupRid)' == 'rhel.7-x64' And '$(Architecture)' == 'Arm64' And '$(UsePortableLinuxSharedFramework)' == 'true' ">rhel.7-aarch64</RuntimeDepsInstallerFileRid>
```

Comment:
Can you help me understand why we are calling this `rhel.7-aarch64` and not `rhel.7-arm64` like [the RID catalog would](https://github.com/dotnet/runtime/blob/89daf9655a6de5610582e3b81b319420e24bb6f5/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json#L2403) (if RHEL 7 + Arm64 appeared there)?

Reply:
I'm not sure if this is right or not, but this seemed to be the pattern the assets from the runtime repo were using.  For example, `dotnet-runtime-6.0.5-cm.2-aarch64.rpm` or `dotnet-runtime-deps-6.0.5-rhel.7-aarch64.rpm`

Reply:
Ah. Thanks!

The RPM names do generally follow the Linux `uname -m` convention (`x86_64` instead of `x64` and `aarch64` instead of `arm64`). Maybe we shouldn't call this variable `,,,Rid`, then?

(As an aside, packages on https://packages.microsoft.com/centos/8/prod/ seem to be randomly mixing `x64` and `x86_64` :confused: )

Reply:
Anyway, sorry for the noise.